huaweicloud/import_huaweicloud_vpc_peering_connection_v2_test.go

```diff
@@ -0,0 +1,29 @@
+package huaweicloud
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+// PASS
```

Comment:
Please help to clean up such test results "// PASS"

Reply:
done